install_config/persistent_storage/persistent_storage_nfs.adoc

```diff
@@ -170,7 +170,7 @@ Ansible playbook] to help you get started.
 By default, SELinux does not allow writing from a pod to a remote NFS server.
 The NFS volume mounts correctly, but is read-only.
 
-To enable writing in SELinux on each node, run:
+To enable writing with SELinux enforcing on each node, run:
```

Comment:
This sentence still feels incomplete to me... how about:

'To enable writing to NFS volumes with SELinux enforcing...'